### ISSUE TITLE
Recaptcha IDs are not unique

### ISSUE THREAD

**Issue by thatbudakguy** [sul-dlss/SearchWorks#3628]
We have two different forms that use the recaptcha that are rendered on some pages (feedback and connection).

By default, recaptcha renders itself with an ID of `g-recaptcha`, so it gets duplicated, which is a WCAG A violation.
